data/health-neighbourhood-london-fields/map-definition.json

```diff
@@ -0,0 +1,564 @@
+{
+    "title": "London Field Health Neighbourhood",
+    "summary": "",
+    "showFullScreenButton": false,
+    "showLegend": true,
+    "showLayersOnLoad": true,
+    "baseStyle": "OSlight",
+    "layers": [
+      {
+        "title": "GP surgeries",
+        "loadToBack": true,
+        "sortOrder": 1,
+        "geoserverLayerName": "health:health_service_gp",
+        "cqlFilter":"INTERSECTS%28geom,%20collectGeometries%28queryCollection%28%27health:health_care_neighbourhood_lf%27,%27geom%27,%27INCLUDE%27%29%29%29",
```

Comment:
We could add the more readable version of the filter
`INTERSECTS(geom,querySingle('health:health_care_neighbourhood','geom','prinx=7'))`

Reply:
Yes, the filters need to be modified. These are not the final ones and will be more user friendly.